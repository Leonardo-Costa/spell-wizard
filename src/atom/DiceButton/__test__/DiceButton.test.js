import React from 'react';
import renderer from 'react-test-renderer';

import {DiceButton} from '../index';

it('renders correctly', () => {
    const tree = renderer.create(<DiceButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });