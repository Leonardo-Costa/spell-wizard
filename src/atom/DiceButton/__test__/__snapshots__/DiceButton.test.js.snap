// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FDC433",
      "borderRadius": 100,
      "height": 40,
      "justifyContent": "center",
      "opacity": 1,
      "width": 40,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#202020",
        "fontSize": 16,
        "fontWeight": "bold",
      }
    }
  />
</View>
`;
